### PR TITLE
i18n(fr) Add french translation for `server-side-rendering.mdx`

### PR DESCRIPTION
**Pull request by McFlyPartages**
#### Description (required)

Add french translation for `server-side-rendering.mdx`

### CODE REVIEW

File: src/content/docs/fr/guides/server-side-rendering.mdx [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+
+- **Points de terminaison d'API** : Créer des pages spécifiques qui fonctionnent comme des points de terminaison API pour des tâches telles que l'accès à la base de données, l'authentification et l'autorisation, tout en gardant les données sensibles cachées au client.
+
+- **Pages protégées** : Restreignez l'accès à une page en fonction des privilèges de l'utilisateur, en gérant l'accès de l'utilisateur sur le serveur.
+
+- **Contenu changeant fréquemment** : Générez des pages individuelles sans avoir à reconstruire votre site de manière statique. Ceci est utile lorsque le contenu d'une page est fréquemment mis à jour.
+
+## Adaptateurs officiels
+
+Astro maintient des adaptateurs officiels pour [NodeJs](https://nodejs.org/), [Vercel](https://vercel.com/), [Netlify](https://www.netlify.com/), et [Cloudflare](https://www.cloudflare.com/).
```
Comment (thomasbnt) [inline src/content/docs/fr/guides/server-side-rendering.mdx:26]:
```suggestion
Astro maintient des adaptateurs officiels pour [Node.js](https://nodejs.org/), [Vercel](https://vercel.com/), [Netlify](https://www.netlify.com/), et [Cloudflare](https://www.cloudflare.com/). 
```

File: src/content/docs/fr/guides/server-side-rendering.mdx [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+
+L'exemple ci-dessous renvoie un message 404 sur une page dynamique après avoir recherché un identifiant dans la base de données :
+
+```astro title="src/pages/[id].astro" {8-11}
+---
+import { getProduct } from '../api';
+
+const product = await getProduct(Astro.params.id);
+
+// No product found
```
Comment (thomasbnt) [inline src/content/docs/fr/guides/server-side-rendering.mdx:246]:
```suggestion
// Aucun produit trouvé
```

File: src/content/docs/fr/guides/server-side-rendering.mdx [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+if (!product) {
+  return new Response(null, {
+    status: 404,
+    statusText: 'Not found'
```
Comment (thomasbnt) [inline src/content/docs/fr/guides/server-side-rendering.mdx:250]:
```suggestion
    statusText: 'Non trouvé'
```

**Review by thomasbnt — CHANGES_REQUESTED** [review]
Little edits like on "Node.js" and two texts in examples. LGTM for the rest of files.

**Comment by McFlyPartages** [comment]
> Little edits like on "Node.js" and two texts in examples. LGTM for the rest of files.

Thanks it's fix

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Good job peeps 🙌

**Pull request merged**